finality-aleph/src/network/build/mod.rs

```diff
@@ -47,10 +49,17 @@ pub struct NetworkOutput<TP: TransactionPool + 'static> {
     pub system_rpc_tx: TracingUnboundedSender<RpcRequest<TP::Block>>,
 }
 
+pub struct SubstrateNetworkConfig {
+    /// Maximum bit-rate in bytes per second of the substrate network (shared by sync, gossip, etc.).
+    pub substrate_network_bit_rate_per_connection: usize,
```

Comment:
Nit: this is a very verbose name, considering the name of the struct it should probably be called just `bit_rate_per_connection`.

Reply:
changed